crypto/fipsmodule/CMakeLists.txt

```diff
@@ -33,6 +33,45 @@ if(ARCH STREQUAL "x86_64")
     x86_64-mont5.${ASM_EXT}
     x86_64-mont.${ASM_EXT}
   )
+
+  if (UNIX AND NOT APPLE)
```

Comment:
s2n-bignum should work on any Unix based system. Currently it doesn't work on Apple OSs because Apple prefixes symbols with an underscore in the compiled object and s2n-bignum's assembly defines function names only without underscore. I'll talk to John about this and if it's the only blocker to get this working on macOS we can easily fix it.

Reply:
In this case it is mostly taste imo. I was thinking in the future when we might add more. But in both cases, we need to maintain a list, so same-same.